apps/backoffice-v2/src/common/components/molecules/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { CalendarIcon } from '@radix-ui/react-icons';
+import { formatDate } from '@/common/utils/format-date';
+import { DateRange } from 'react-day-picker';
+import { addMonths } from 'date-fns';
```

Comment:
Considering we use `dayjs` see if we can achieve the same using `dayjs` and not `date-fns`. 🙏🏼